### PR TITLE
session: add transaction id in general log.

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @zimulala PTAL

File: session.go [hunk 1]
```diff
@@ -1575,6 +1575,6 @@ func logStmt(node ast.StmtNode, vars *variable.SessionVars) {
 
 func logQuery(query string, vars *variable.SessionVars) {
 	if atomic.LoadUint32(&variable.ProcessGeneralLog) != 0 && !vars.InRestrictedSQL {
-		log.Infof("[%d] %s", vars.ConnectionID, query)
+		log.Infof("[con:%d][txn:%d] %s", vars.ConnectionID, vars.TxnCtx.StartTS, query)
```
Comment (zimulala) [inline session.go]: `StartTS` sometimes maybe 0?

Comment (coocood) [reply session.go]: I think it's ok, we care more about multiple statements in a transaction.

**Pull request merged**